### PR TITLE
Replace repetitive actions with Action Groups in AdminDeleteConfigurableChildProductsTest

### LINKED ISSUE #33080: [Issue] Replace repetitive actions with Action Groups in AdminDeleteConfigurableChildProductsTest
This issue is automatically created based on existing pull request: magento/magento2#31791: Replace repetitive actions with Action Groups in AdminDeleteConfigurableChildProductsTest

---------

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)
Run the test in the local environment and verified that it run successfully.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)
Run the test in the local environment and verified that it run successfully.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#33080: Replace repetitive actions with Action Groups in AdminDeleteConfigurableChildProductsTest

### CODE REVIEW

**Comment by kate-kyzyma** [comment]
@magento run all tests

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeLabelVisibleActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertProductAttributeLabelVisibleActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeLabelVisibleActionGroup.xml]: Please, correct the naming according to the naming convention (see section 11.3.5 of the [guidelines](https://devdocs.magento.com/guides/v2.4/coding-standards/technical-guidelines.html#113-functional-testing)).

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeOptionVisibleActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertProductAttributeOptionVisibleActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeOptionVisibleActionGroup.xml]: Please, correct the naming. Check my previous comment for details

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @kate-kyzyma. Good job! 

Please, check my comments for a couple of minor adjustments. 

Thanks!

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeOptionVisibleActionGroup.xml (hunk 2 above)
Comment (kate-kyzyma) [reply app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeOptionVisibleActionGroup.xml]: Done!

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeLabelVisibleActionGroup.xml (hunk 1 above)
Comment (kate-kyzyma) [reply app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeLabelVisibleActionGroup.xml]: Done!

**Comment by kate-kyzyma** [comment]
Hello, @rogyar!
Thank you for reviewing my test. I have updated naming for ActionGroups.
Please, check.

Thanks.

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8704](https://jira.corp.magento.com/browse/ENGCOM-8704) has been created to process this Pull Request

**Comment by kate-kyzyma** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by kate-kyzyma** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Integration Tests

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by kate-kyzyma** [comment]
@magento run Unit Tests

**Comment by engcom-Oscar** [comment]
@magento create issue

**Comment by dmitriyprime** [comment]
:heavy_check_mark:  QA passed

**Pull request merged**